File: Easy/problem_2/sol.cs
using System;

class Program
{
    static void Main()
    {
        PrintEven(100);
        PrintOdd(100);
    }

    public void PrintEven(int n)
    {
        for (int i = 0; i < n; i++)
        {
            if (i % 2 == 0)
            {
                Console.WriteLine(i);
            }
        }
    }

    public void PrintOdd(int n)
    {
        for (int i = 0; i < n; i++)
        {
            if (i % 2 == 1)
            {
                Console.WriteLine(i);
            }
        }
    }
}

File: Easy/problem_2/sol.py
def print_even(n):
    for i in range(0, n):
        if i % 2 == 0:
            print(i)


def print_odd(n):
    for i in range(0, n):
        if i % 2 != 0:
            print(i)


print_even(100)
print_odd(100)
